third_party/imx-secure-enclave/BUILD.gn

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2023 Project CHIP Authors
+# Copyright 2023 NXP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+declare_args() {
+  rom = 1
```

Comment:
create better names: these are global paramebers. You probably need to prefix like imx_secure_enclave_.... `rom/fmw` is not clearly readable, major_ver/minor_ver/plat_path/common_path are not specific enough (these will go into the global namespace).